certified-connectors/Seismic Configuration/readme.md

```diff
@@ -0,0 +1,51 @@
+# Seismic Configuration
+Seismic configuration connector provides set of actions required for adding or updating content properties, fetching teamsites and updating email opt-outs.
+
+## Publisher: Seismic​
+
+## Prerequisites
+You must have user created in Seismic tenant.
+
+## Supported Operations
+### Get Content Property Values
+Provides the list of content property values for a given content property. This will only return values for properties that have a defined list of values. Properties which have no defined domain of values and are filled in freeform by users are not provided here.
+
+### Add Content Property Values
+Add one or more content property values to an existing content property. Property values cannot be added to properties where an existing domain of values does not exist. Property values are case sensitive.#### Warning:Adding a domain of values to an existing property that does not have an domain of values can result in a loss of data. If this is desired please log into Seismic and create a domain of values to enable this functionality.
```

Comment:
@rkushvaha all files should be free of grammatical and spelling errors. in the description you have "...that does not have an domain of values..." please change to "...that does not have **a domain** of values...".